### PR TITLE
Doc/fix/durabilityref2 - New version of durability referencing

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Cherry picked from broken branch.

### CODE REVIEW

File: product_docs/docs/pgd/5/durability/commit-scopes.mdx [hunk 1]
```diff
@@ -4,11 +4,11 @@ title: Commit scopes
 
 Commit scopes give applications granular control about durability and consistency of EDB Postgres Distributed.
 
-A commit scope is a named rule that describes the behavior of COMMIT replication. The actual behavior depends on whether a commit scope uses [group commit](group-commit), [Commit At Most Once](camo), [lag control](lag-control), or combination of these.
+A commit scope is a named rule that describes the behavior of COMMIT replication. The actual behavior depends on whether a commit scope uses [Group Commit](group-commit), [Commit At Most Once](camo), [lag control](lag-control), or combination of these.
 
 ## Configuration
 
-To use group commit, first define a commit scope, which determines the PGD nodes involved in the commit of a transaction. Once a scope is established, you can configure a transaction to use group commit as follows:
+To use Group ommit, first define a commit scope, which determines the PGD nodes involved in the commit of a transaction. Once a scope is established, you can configure a transaction to use Group Commit as follows:
```
Comment (mwanner) [inline product_docs/docs/pgd/5/durability/commit-scopes.mdx:11]: Group Omit? :-)

File: product_docs/docs/pgd/5/durability/lag-control.mdx [hunk 2]
```diff
@@ -11,7 +11,7 @@ time or chronically in high availability applications, then
 organizational objectives related to disaster recovery or business
 continuity plans might not be satisfied.
 
-The replication lag control (RLC) feature is designed to regulate this
+The replication Lag Control (RLC) feature is designed to regulate this
```
Comment (mwanner) [inline product_docs/docs/pgd/5/durability/lag-control.mdx:14]: Should "Replication" also start with an uppercase letter in this case, i.e. when we abbreviate it to RLC? It seems later on, we don't ever use RLC, but only use "Lag Control" (w/o the R word).

File: product_docs/docs/pgd/5/durability/legacy-sync.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
+---
+title: Legacy synchronous replication using PGD
+navTitle: Legacy synchronous replication
+---
+
+!!! Warning Important
+Consider using [Group Commit](group-commit) instead.
+!!!
```
Comment (mwanner) [inline product_docs/docs/pgd/5/durability/legacy-sync.mdx:8]: I wonder if this still is true. Or if we can refer to the commit scope based Sync Rep option, instead. Do we have that documented by now?

I guess it's good enough for this PR and we can revisit Sync Rep independent of this PR.

**Review by mwanner — DISMISSED** [review]

File: product_docs/docs/pgd/5/durability/commit-scopes.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/commit-scopes.mdx:11]: Ooopsy.

File: product_docs/docs/pgd/5/durability/legacy-sync.mdx (hunk 3 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/legacy-sync.mdx:8]: We don't have it documented yet, was waiting on details from Petr. The next iteration on this doc would be to bring in Sync Rep.

File: product_docs/docs/pgd/5/durability/lag-control.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/lag-control.mdx:14]: I think just calling it Lag Control might cover it and then we don't need the acronym.

**Pull request closed without merging**